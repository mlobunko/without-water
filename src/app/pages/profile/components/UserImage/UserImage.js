import React from 'react';

import { StyledUserImage } from '../../styles';

export const UserImage = () => <StyledUserImage />;

UserImage.displayName = 'UserImage';

export default UserImage;
