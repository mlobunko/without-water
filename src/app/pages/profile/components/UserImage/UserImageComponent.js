import React from 'react';

export const UserImageComponent = () => <div className="image-user" />;
