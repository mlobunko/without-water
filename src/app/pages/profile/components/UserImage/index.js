import { UserImageComponent as UserImage } from './UserImageComponent';

export default UserImage;
