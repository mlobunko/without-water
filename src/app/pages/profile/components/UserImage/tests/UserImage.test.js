import React from 'react';
import { shallow } from 'enzyme';

import { UserImage } from '../UserImage';

describe('render', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<UserImage />);
    expect(wrapper).toMatchSnapshot();
  });
});
