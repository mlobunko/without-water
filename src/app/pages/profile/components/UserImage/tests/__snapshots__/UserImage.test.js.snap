// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `<StyledUserImage />`;
